shared/router-v2/router.desktop.tsx

```diff
@@ -86,24 +69,32 @@ const appTabsInnerOptions = {
   tabBarShowLabel: Kb.Styles.isTablet,
   tabBarStyle: Common.tabBarStyle,
 }
-const AppTabsInner = () => {
-  // so we have a stack per tab
-  const tabStacks = React.useMemo(
-    () => Tabs.desktopTabs.map(tab => <Tab.Screen key={tab} name={tab} component={makeTabStack(tab)} />),
-    []
+
+const TabStack = createNativeStackNavigator()
+const TabStackNavigator = React.memo(function TabStackNavigator(p: {route: {name: string}}) {
+  const tab = p.route.name as DesktopTabs
+  const tabScreens = React.useMemo(
```

Comment:
even though this is memoed the parent will force this to render due to navigation context changing, so we must useMemo these screens else they get remade